src/core/pipelines/manifest/index.ts

```diff
@@ -103,10 +104,20 @@ export default function createManifestPipeline(
         arg.type === "response"
       ),
 
-      mergeMap(({ value }) => parser({ response: value, url })),
-
-      map(({ manifest }) : Manifest => {
-        return new Manifest(manifest, warning$, transport.options);
+      mergeMap(({ value }) => {
+        const { sentTime } = value;
+        return parser({ response: value, url }).pipe(
+          map(({ manifest: parsedManifest }) => {
+            const manifest = new Manifest(parsedManifest, warning$, transport.options);
+            const manifestFetchingDuration = sentTime ?
```

Comment:
I've two main problems with the fact that we communicate a `manifestFetchingDuration`:
  - it is not a fetching duration but a fetch + parse one
  - the time between this operation and the usage of it could be extended in future evolutions without us expecting any problem with that

If I understood it correctly this operation should be the closest possible to `performance.now` at the time the corresponding timeout is started.
Why not just communicating the `requestTime`. It might even make more sense as a `manifestUpdate` event, as it can be used to sort chronologically such updates.